omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
I would ensure `file_cache_path` is also set:

```ruby
file_cache_path "/var/opt/supermarket/local-mode-cache"
```

These two config options usually get changed in lock step.